### PR TITLE
Migrate to VitePress

### PR DESCRIPTION
**Pull request by spencerwooo**
- [x] Basic setup
- [x] Remove legacy VuePress configurations
- [x] Fix dead links
- [x] Update screenshots
- [ ] ~~Update obsolete documentation~~

### CODE REVIEW

**Comment by spencerwooo** [comment]
Breaking changes:

- `pnpm` is used instead of `yarn`, build commands in Netlify should be updated.
- ...

**Comment by spencerwooo** [comment]
CleanShot 2022-12-17 at 11 54 35@2x

Ready to be deployed. cc @fky2015 

等合并后，我考虑重写一部分文档，包括移除没有参考价值的视频部分。

File: wiki/guide/intro.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+![Screenshot of Undergraduate Thesis template](../assets/screenshot-undergrad.png)
 
 BIThesis 为各位提供了基于北京理工大学计算机学院给出的「北京理工大学本科生毕业论文 · 开题报告」与北京理工大学教务部给出的「北京理工大学本科生毕业设计 · 论文模板」的 LaTeX 样版，基于 BIThesis 的 LaTeX 模板，你可以在不必过多在意论文格式的基础上，专注于学术研究、项目实现，从而顺利完成你的项目。
 
 ## 开始阅读文档
 
 这里是 BIThesis 的项目 Wiki，提供了 LaTeX 的基本安装方法、LaTeX 项目的使用方法、BIThesis 的编译方法……你可以按照下面的流程图所示的顺序来下载与使用 BIThesis：
 
-![](https://i.loli.net/2020/03/27/mV3kTsnCDNWwlOU.png)
+TODO: Update this diagram.
```
Comment (fky2015) [inline wiki/guide/intro.md:21]: Is this intentional?

**Comment by fky2015** [comment]
And the intro page of `Video` is unavailable, is this intentional too?

File: wiki/guide/intro.md (hunk 1 above)
Comment (spencerwooo) [reply wiki/guide/intro.md:21]: 预期是合并后然后重写部分文档，再更新这个路线图。

**Comment by spencerwooo** [comment]
> And the intro page of `Video` is unavailable, is this intentional too?

Forgot about this page in the sidebar config. Latest commit should resolve this no?

**Review by fky2015 — APPROVED** [review]

**Pull request merged**